src/Attribute/SplitStream.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Attribute;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+final class SplitStream
```

Comment:
I don't like the name of this attribute. How about:

* RebootStream
* RestartStream

?

Reply:
Or "NewStart" ?

Reply:
Well, i think all the proposed names are not good either tbh. 

* RebootStream feels off here
* RestartStream feels more like reloading
* NewStart meh - i dont know if the user can understand this

We should first think about how we want the feature to be in the future. Do we want to stay with the `achieved` bool flag or do we want to *really* split / restart the stream? 

For me SplitStream is the most accurate, as we are already using the stream term in some places in docs and code. I think this is the most understandable name from the 4 options right now. But we can put more thought about the name for sure.

Reply:
Renamed it on the message to be more clear in the message context what the flag is supposed to do. Will resolve this for now with that.